docs/operations_guide/Flink.md

```diff
@@ -271,7 +271,7 @@ Diagnosing most of the problems below requires access to:
 | Checkpoints are failing                                                                     | Check jobmanager logs and/or Flink console                                                                                                                                                              |
 | Redeploy of scenario times out                                                              | Check jobmanager logs and/or Flink console                                                                                                                                                              |
 | `State is incompatible, please stop process and start again with clean state` during deploy | <ul><li>Check if Nussknacker has access to savepoints</li><li>Analyze if new state was added - if this is the case probably cancel before deploy is needed (to get rid of incompatible state)</li></ul> |
-| Aggregate events are not emitted by aggregate in time windows nodes                         | Idle source (Kafka topic) or idle Kafka partition confuse Flink's watermark mechanism. To avoid this problem ensure that each Kafka source partition continually gets events |
+| Aggregate events are not emitted by aggregate in time windows nodes                         | <br> Idle source (Kafka topic) or idle Kafka partition confuse Flink's watermark mechanism. To avoid this problem ensure that each Kafka source partition continually gets events </br>  Alternatively change [idleTimeout](https://nussknacker.io/documentation/docs/integration/KafkaIntegration/#configuration-for-flink-engine) configuration to ensure that idle source emits watermarks at acceptable intervals.  |
```

Comment:
```suggestion
| Aggregate events are not emitted by aggregate in time windows nodes                         | Idle source (Kafka topic) or idle Kafka partition confuse Flink's watermark mechanism. To avoid this problem ensure that each Kafka source partition continually gets events.</br>Alternatively change [idleTimeout](https://nussknacker.io/documentation/docs/integration/KafkaIntegration/#configuration-for-flink-engine) configuration to ensure that idle source emits watermarks at acceptable intervals.  |
```

Reply:
applied suggestion